DiscordBot/Settings/Deserialized/MessagePresets.cs

```diff
@@ -0,0 +1,7 @@
+using System.Collections.Generic;
```

Comment:
You can have both of your MessagePreset classes in the same file. If you look at other Deserialized files, you can see it in action.